airflow/include/astronomermonitoringdag.go

```diff
@@ -0,0 +1,33 @@
+package include
+
+import "strings"
+
+var MonitoringDag = strings.TrimSpace(`
```

Comment:
We add this monitoring DAG on runtime when dag deploy is enabled to make sure its consistent across all deployments.